project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,20 @@
+import sbt.Setting
+import scoverage.ScoverageKeys.*
+
+object CodeCoverageSettings {
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    "Reverse.*",
```

Comment:
Can you please remove this as it is redundant. Also, handlers can be removed if you want to. There are many examples of this test e.g. [here](https://github.com/hmrc/binding-tariff-trader-frontend/blob/main/test/handler/ErrorHandlerSpec.scala). It seems some tests are missing for these two in [templates](https://github.com/hmrc/advance-valuation-rulings-frontend/tree/main/app/views/templates). Since it is not excluded, please add tests, thanks.

Reply:
As not sure both layouts are required and think we were going to have a ticket for fixing the deprecated view layout warning, perhaps should solve this there? Can add an exclusion for it now?

Reply:
Yh tests for Layout is required regardless as this a template used across all views so that is needed. On the other one I have seen a full layout in some of our services so that might also be required and can be done via TemplateOverrides from play frontend hmrc. Personally, I would like tests for this and we can deal with the other  issues. If tests are not written for this then that's fine but yh don't want to exclude it so it is visible. Just means we will need to remember to write tests for this when the work is eventually done.